assets/js/screens/Dashboard/index.js

```diff
@@ -0,0 +1,32 @@
+import Dropdown from '../../components/Dropdown';
+
+const SELECTOR = {
+  screen: 'body.dashboard',
+  uploadReportButton: '.upload-report .form-report__button'
+};
+
+class DashboardScreen {
+  /**
+   * Initializer
+   */
+  constructor() {
+    this.uploadReportButton = document.querySelector(SELECTOR.uploadReportButton);
+
+    this._setup();
+  }
+
+  /**
+   * Component bootstrapping actions.
+   */
+  _setup() {
+    this.uploadDropdown = new Dropdown(this.uploadReportButton);
```

Comment:
How about we move "Dropdown" setup to the initializer then use `data-toggle="dropdown"` as a selector? So we don't need to initialize this in every screen.

Reply:
I don't really follow, don't we still need to initialize on each screen there's a drop-down anyways? Since this JS file is related to the dashboard screen only. 🤔 

I've removed the dashboard component and move the setup here in fe21644f6ef4774805c7d80baad921e1fd568880, 
but I'm also thinking we could also do something like we could export some default class selector from the Dropdown component as well. 

I'm fine with either, what do you think? @hanam1ni

Reply:
You can check `Initializers` in [this convention](https://nimblehq.co/compass/development/code-conventions/javascript/#front-end-js) so we only need a single file to initialize the application-wide component.

Reply:
Oh, I gotcha. I've done that in eaa050b81c51144ae361709f585856e68a57e41e 👍